mitiq/calibration/settings.py

```diff
@@ -177,6 +177,7 @@ def __init__(
         num_qubits: int,
         circuit_depth: int,
         strategies: List[Dict[str, Any]],
+        circuit_seed: Optional[int] = None,
```

Comment:
`circuit_seed` should be added to the docstring of `Settings`

Reply:
Agreed, done.